core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
Error highlighting is not done by changing the background, see [mocks ](https://drive.google.com/drive/folders/0B_8dp8q8tZvqSWtqRkpTY0djckU) please.